docs/source/user/interface.rst

```diff
@@ -161,6 +161,17 @@ Toggle Simple Interface mode using the View menu:
 When you leave Simple Interface mode, the original layout of the main
 area is restored.
 
+Searching
+---------
+
+JupyterLab has an intelligent built-in search plugin for finding text within a
+notebook or other document, which uses the `Ctrl-F` shortcut by default.
+
+Your browser's `find` function will give unexpected results because it doesn't have
+access to the full content of a document, but you can still use your browser find
+function from the browser menu if you want, or you can disable the intelligent search
```

Comment:
```suggestion
function from the browser menu if you want, or you can disable the built-in document search
```

Reply:
This has been incorporated in a separate commit.